source/frontend/src/features/properties/map/acquisition/common/models.ts

```diff
@@ -76,6 +77,7 @@ export class AcquisitionOwnerFormModel {
       id: this.id,
       rowVersion: this.rowVersion,
       acquisitionFileId: this.acquisitionFileId,
+      isPrimaryContact: this.isPrimaryContact === 'true' ? true : false,
```

Comment:
although trivial, there are functions that perform the string 'true' to true conversion in our common libraries

Reply:
Updated